src/view.ts

```diff
@@ -472,6 +476,55 @@ export class WriteableStickyNote extends AbstractStickyNote {
     if (compare_tags(tags, this.note!.tags)) return;
     this.change("tags", tags);
   }
+
+  on_text_inserted(buffer: Gtk.TextBuffer, loc: Gtk.TextIter, text: string, length: number) {
+    if (text === '\n') {
+      const start_iter = loc.copy();
+      start_iter.backward_char();
+      start_iter.set_line_offset(0);
+
+      const end_iter = start_iter.copy();
+      end_iter.forward_chars(2);
+      const chars = buffer.get_text(start_iter, end_iter, false);
+
+      const simple_regex_pattern = /[-+*]\s$/;
```

Comment:
You might need to test for sentences starting with the given characters. Something like this, although I'm not sure

```suggestion
      const simple_regex_pattern = /^[-+*]\s$/;
```

Reply:
Yes, you are right, strangely when testing the app it didn't give any problem, going to test it with the changes anyway.